common.blocks/version-selector/version-selector.js

```diff
@@ -46,7 +46,7 @@ modules.define('version-selector', [ 'i-bem__dom', 'querystring' ], function(pro
 
                         transport = d.createElement('script');
                         transport.id = TRANSPORT_ID;
-                        transport.src = URL + val + FILE;
+                        transport.src = URL + hash + FILE;
```

Comment:
For example, we can not append anything to body, and write `console.error` at least…